Vox Wintercoats

Davyei opened the pull request:
<!--
You are making this pull request for the Moffstation fork of Space Station 14.

Please be sure to follow general guidelines for upstream PRs, but also be sure to follow the Moffstation guidelines.
Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline
See the Harmony contributing guidelines for an example on what we want: https://github.com/ss14-harmony/ss14-harmony/blob/master/CONTRIBUTING.md
-->

## About the PR
<!-- What did you change? -->
- Makes it so the vox sprites are used
- Adds missing vox sprites & vox sprites from moffstation outfits also
- Edits some of the existing coloured winter coats to be more appealing to wear.
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Vox drip!
## Technical details
<!-- Summary of code changes for easier review. -->

## Media
<!-- Attach media if the PR makes in-game changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="640" height="433" alt="Screenshot 2025-12-13 151802" src="https://github.com/user-attachments/assets/d3755481-8870-4fe0-a554-b6379a0fb022" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Upstream Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html) as well as the [Moffstation Contributing Guidelines](https://github.com/moff-station/moff-station-14/blob/master/CONTRIBUTING.md).
- [X] I have properly sectioned my changes into fork namespaces, and/or followed proper guidelines for modifying upstream files.
- [X] I have added media to this PR or it does not require an in-game showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:CL:
- add: vox wintercoats sprites

Davyei committed:
Update wintercoats.yml

adds the vox sprites states to all winter coats except centcom, hop, hos, jani, web, ward, rd, robo, cmo, chem, viro, gene, qm, chef, syndi, syndicap which I will make the vox sprites for soon

Davyei committed:
adds vox sprites

Davyei committed:
extra thingies

should be maybe everything sorted hopefully

Davyei committed:
update to have engi resprites

Davyei committed:
Merge branch 'refs/heads/master' into vox-wintercoats

Davyei committed:
existing moff station vox wintercoat sprite implementation

Davyei committed:
Sec coat sprites

Centronias reviewed (changes requested):
"CoatyBits" would be a great moff snack name.

Davyei committed:
Moving house

I packed up all my things and moved my skates to the Moffstation namespace

Davyei committed:
forgot to take my things out

woops, things should be fine now

Davyei committed:
missed two. :P

Davyei committed:
cmo coat & sneaky .rsi changes

deleted the wrong cmo state from the .json & my changes for another pr tried to sneak in
